Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,220 @@
+File is an instance_of generic dependent continuant
+File 1 Turtle implements content 1
+File 2 XML implements content 1
+Files 1 and 2 are output of Computer 1
+Computer 1 is bearer of File 1 and File 2
+Computer 1 is an instance_of material entity
+Computer 1 sends output to Computers 2 and 3
+File 1 is input of Computer 2
+Computer 2 receives input from Computer 1
+Computer 2 is an instance_of material entity
+File 2 is input of Computer 3
+Computer 3 receives input from Computer 1
+Computer 3 is an instance_of material entity
+Computer 1 2 and 3 are part of Network 1
+Network 1 is an instance_of Network
+Network is_a object aggregate
+
+
+6. Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  Generically Dependent Continuant
+  (b) Chair of the UB Philosophy Department
+  Role
+  (c) SARS-CoV-2
+  Object
+  (d) Mexico City
+  Object Aggregate
+  (e) The trunk of a minivan
+  Fiat Object Part
+  (f) Occupation
+  Role
+  (g) Ocean
+  Object Aggregate
+  (h) Lake
+  Object Aggregate
+```
+
+7. True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  True. A material entity has some portion of matter as proper or improper continuant part. These parts don't need to be material. For example thoughts are immaterial but can be understood as a part of the material person. 
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  False. An immaterial entity is immaterial, so it may be concretized in a material entity or otherwise but may not have it as part. For example, a thought may be an immaterial entity which has a material entity as a subject, but that subject is either not material (i.e is a representation of a material thing) or is not a part of that thought.
+  (c) An organization may have another organization as part.
+  True. Organizations break off separate organizations to perform different functions all of the time. A charity organization for finding housing for the homeless might break off local and regional organizations to better serve people at various locations for example.
+  (d) An organization may have no members as part. 
+  True. Organizations can continue their existence even after losing all members for a time. For example, the Soviet Union continued to exist for a time even after all member states left the organization. It's continuation, then, depended not on its members but instead on other factors like the organizational structure and rules.
+  (e) Any site is partially bounded by some instance of Material Entity.
+  True. All examples of sites used in the BFO Protege consist of gaps between material entities. Gaps such as this, while separate from their surrounding material entities, will be partially defined by those material entities. For example, the Grand Canyon is specifically a gap defined by surrounding cliffs.
```

Comment:
(e) False. A site, according to its definition in BFO, may be a **three-dimensional immaterial part** of an immaterial entity (partially or wholly) bounded by a material entity. In this case, the site in question may be itself not partially bounded by a material entity.

Reply:
Should be fixed to reflect this point.